<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipePanel</class>
 <widget class="QWidget" name="RecipePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="drinkNameTextField">
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteRecipeButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>img</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addImageButton">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Set Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="saveRecipeButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>Молоко</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="milkAmountTextField">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>мл</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>Сливки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="creamAmountTextField">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>мл</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>Вода</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="waterAmountTextField">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>мл</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>Кофе</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="coffeeAmountTextField">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>35</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>гр</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
